pkg/plugins/golang/v3/scaffolds/internal/templates/api/group.go

```diff
@@ -30,12 +31,19 @@ type Group struct {
 	machinery.MultiGroupMixin
 	machinery.BoilerplateMixin
 	machinery.ResourceMixin
+	machinery.RepositoryMixin
 }
 
 // SetTemplateDefaults implements file.Template
 func (f *Group) SetTemplateDefaults() error {
 	if f.Path == "" {
-		if f.MultiGroup {
+		if f.Resource.Path != "" {
```

Comment:
Why the path for a project that has no multi-group layout enabled is not the default one?
Could you share what are the commands/steps required to reproduce the scenario which motivates you to raise this PR?

Reply:
I'm looking here if the path was configured in the configuration file, otherwise I revert to the default values (=not multi group)